nb/api/bend.nb

```diff
@@ -13,7 +13,7 @@ See: [remesh](#https://raw.githubusercontent.com/jsxcad/JSxCAD/master/nb/api/rem
 `;
 
 Box(40, 1, 5)
-  .and(bend(20))
+  .And(y(20), bend(20))
```

Comment:
Does capital And do anything different?

Reply:
The capital forms only use the input shape for context.

So this produces two versions of Box(40, 1, 5) -- one translated, and one bent.

The lowercase form would produce three versions -- the original, the translated, and the bent.